rotkehlchen/accounting/transactions.py

```diff
@@ -25,15 +26,15 @@ def evm_events_iterator(
     Takes associated event as an argument to be able to log it in case of errors.
     """
     for event in events_iterator:
-        if not isinstance(event, EvmEvent):
+        if not isinstance(event, EvmEvent) and not isinstance(event, HistoryEvent):  # TODO: undo. Is a temporary hack  # noqa: E501
             log.error(
                 f'At accounting for tx_event {associated_event.notes} with hash '
                 f'{associated_event.tx_hash.hex()} we expected to take an additional '
                 f'event but found a non history base entry event',
             )
             return
 
-        yield event
+        yield event  # type: ignore[misc]
```

Comment:
Maybe you can rename this function? Is used in two places and even being this a hack is really confusing to follow if things are incorrect due to naming